### PR TITLE
Add info about the happy hacking day.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

The happy hacking day is only documented in our private wiki, but there is no reason to keep it private.

Changes
=======

This is just a copy paste from the private wiki.

How to try and test the changes
===============================

reviewers: @lesingerouge @dumol 

check that changes make sense

feel free to edit the document and add more information / fix typos

Thanks!

### CODE REVIEW

**Comment by dumol** [comment]
**changes-approved**

File: content/defects-tech-debt.rst [hunk 1]
```diff
@@ -141,6 +145,37 @@ re-usable component itself so that it provide a clean re-usable API.
 We are not blocking a release due to a tech-debt.
 
 
+Happy Hacking Day
+=================
+
+One day each month is reserved for working on whatever task you want.
+
+This is call the 'Happy Hacking Day' or 'I work on whatever I want day'.
```
Comment (lesingerouge) [inline content/defects-tech-debt.rst]: "called"

File: content/defects-tech-debt.rst [hunk 2]
```diff
@@ -141,6 +145,37 @@ re-usable component itself so that it provide a clean re-usable API.
...
+
+This is the first Wednesday of each month.
+
+The idea is that we always have top priority task on which we work on a regular basis.
```
Comment (lesingerouge) [inline content/defects-tech-debt.rst]: "tasks"

File: content/defects-tech-debt.rst [hunk 3]
```diff
@@ -141,6 +145,37 @@ re-usable component itself so that it provide a clean re-usable API.
...
+
+This is why minor/small task will never get top priority.
```
Comment (lesingerouge) [inline content/defects-tech-debt.rst]: "tasks"

File: content/defects-tech-debt.rst [hunk 4]
```diff
@@ -141,6 +145,37 @@ re-usable component itself so that it provide a clean re-usable API.
...
+
+If there is a minor/small task which has big impact on our day to day work,
```
Comment (lesingerouge) [inline content/defects-tech-debt.rst]: "has a big"

**Comment by lesingerouge** [comment]
changes-approved

**Pull request merged**